dtfs-central-api/src/v1/controllers/utilisation-report-service/tfm-put-utilisation-report-status.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from 'express';
+import { Collection, DeleteResult, UpdateResult } from 'mongodb';
+
+import db from '../../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../../constants/dbCollections';
+import { PutReportStatusRequestBody } from '../../../types/utilisation-report-status';
+import { setReportStatusByReportId, setReportStatusByReportDetails } from '../../../services/repositories/utilisation-report-status-repo';
+
+export const putUtilisationReportStatus = async (req: Request<{}, {}, PutReportStatusRequestBody>, res: Response) => {
+  try {
+    const { reportsWithStatus, user } = req.body;
```

Comment:
Does `user` exist on `req.session`?

Reply:
Yeh that was what I thought as well - I already made that change in a local commit